templates/content.mustache

```diff
@@ -9,15 +9,9 @@
     		{{> title.mustache}}
     	{{/unless}}
 
-      <!--
-      <ul class="on-this-page fixed open">
-        <li><a href="#">Monkeys</a></li>
-        <li class="active"><a href="#">Fish</a></li>
-        <li><a href="#">Cats</a></li>
-        <li><a href="#">Dogs</a></li>
+      <ul class="on-this-page">
```

Comment:
Because doing something in JS or a template is "worse than" in a stylesheet generally speaking. 

There might be cases where you want that ul when empty to show some sort of empty message. 

Sent from my iPhone

> On Jul 14, 2016, at 10:34 PM, Tom Greever <notifications@github.com> wrote:
> 
> In templates/content.mustache:
> 
> > @@ -9,15 +9,9 @@
> >      		{{> title.mustache}}
> >      	{{/unless}}
> >  
> > -      <!--
> > -      <ul class="on-this-page fixed open">
> > -        <li><a href="#">Monkeys</a></li>
> > -        <li class="active"><a href="#">Fish</a></li>
> > -        <li><a href="#">Cats</a></li>
> > -        <li><a href="#">Dogs</a></li>
> > +      <ul class="on-this-page">
> Even if it can't be seen on the page, an empty container is wasteful. But depending on any CSS which applies to the container in general, it could easily have margins, padding, even a border. ULs in particular usually have default padding and margins. I don't know why devs seem to avoid putting conditionals around stuff like this, but I have seen it cause problems before. It's a best practice, imo, to not print HTML that isn't needed
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>